### PR TITLE
blergh

### PR DESCRIPTION
**Pull request by chrismccord**

### CODE REVIEW

File: source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Using Elixir 1.5's open command with terminal emacs"
```
Comment (duggiefresh) [inline source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md]: Perhaps capitalize `Emacs` throughout the post

File: source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+wasn't enough for `open` to launch the buffer inside a single emacs
+instance, since I often have half a dozen tmux sessions, each with
+their own emacs instance for the project and iex sessions running. I
+needed a solution that allowed `open` to target `ELIXIR_EDITOR` at my
+current project's emacs process. And down the rabbit hole I went.
+
+To make it happen, I made use of emacs' built in client/server feature
+where emacs can start a "server" and `emacsclient` can attach or
+interact with it from elsewhere. For the "current project" target, I
+first check for an active git repo, and fallback to the current
```
Comment (duggiefresh) [inline source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md]: capitalize `Git`

File: source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Using Elixir 1.5's open command with terminal emacs"
+social: true
+author: Chris McCord
+twitter: "chris_mccord"
+github: chrismccord
+summary: "Configure Elixir's open command to work inside your terminal emacs editors"
+published: true
+tags: elixir, emacs
```
Comment (brzpegasus) [inline source/posts/2017-08-24-elixir-open-command-with-terminal-emacs.md]: Can you also tag with "engineering"?

**Comment by chrismccord** [comment]
Done!

**Pull request merged**